File: fizzbuzz.cpp
#include <iostream>
using namespace std;

int main () {

    for (int i = 1; i < 101; ++i)
    {
        if (i % 3 == 0 and i % 5 == 0){
            cout << "fizzbuzz" << "\n";
        }
        else if ( i % 3 == 0){
            cout << "fizz" << "\n";
        }
        else if (i % 5 == 0){
            cout << "buzz" << "\n";
        }
        else {
            cout << i << "\n";
        }
    }
}

File: fizzbuzz.py
def fizz_buzz(n):
    fin_list = []
    for number in range(1, n + 1):
        if number % 3 == 0 and number % 5 == 0:
            fin_list.append('fizzbuzz')
        elif number % 3 == 0:
            fin_list.append('fizz')
        elif number % 5 == 0:
            fin_list.append('buzz')
        else:
            fin_list.append(number)
    return fin_list